src/lib/cacheMiddleware.ts

```diff
@@ -1,76 +0,0 @@
-import { Prisma } from '@app/generated/centralLedger';
-
-interface Cache {
-  setValue: (key: string, value: object) => void;
```

Comment:
What is the reason we removed this cache?

Reply:
This middleware was being used by the central ledger and had no other use in the service, so this was removed